client/src/components/AddReviewModal.jsx

```diff
@@ -33,44 +61,291 @@ class AddReviewModal extends React.Component {
+      characteristics[this.props.characteristics[characteristic].id] = undefined;
+      tempCharacteristics[characteristic] = undefined;
+    }
+    this.setState({
+      characteristics: characteristics,
+      tempCharacteristics: tempCharacteristics
+    });
   }
 
 
   render () {
+
+    // Handles dynamically rendering characteristics distinct to each item
+    let characteristics = [];
+    let characteristicNames = Object.keys(this.props.characteristics);
+    for (let i = 0; i < characteristicNames.length; i++) {
+      let characteristicName = characteristicNames[i];
+      let characteristic = (
+        <div key={i} className="formCharacteristic">
+          <label htmlFor={characteristicName} className="formCharacteristicName">{characteristicName + ' '}</label>
+          <span className="formChosenCharacteristicDescription">{this.state.tempCharacteristics[characteristicName] ? ' - ' + this.state.tempCharacteristics[characteristicName] : undefined}</span>
+          <div className="formCharacteristicBtnContainer">
+            <input type="radio" name={characteristicName} value={0} className="characteristicInput"
+              onChange={e => this.handleChange(e)} 
+              checked={this.state.tempCharacteristics[characteristicName] === this.characteristicDescriptions[characteristicName][0]}></input>
+            <input type="radio" name={characteristicName} value={1} className="characteristicInput"
+              onChange={e => this.handleChange(e)} 
+              checked={this.state.tempCharacteristics[characteristicName] === this.characteristicDescriptions[characteristicName][1]}></input>
+            <input type="radio" name={characteristicName} value={2} className="characteristicInput"
+              onChange={e => this.handleChange(e)} 
+              checked={this.state.tempCharacteristics[characteristicName] === this.characteristicDescriptions[characteristicName][2]}></input>
+            <input type="radio" name={characteristicName} value={3} className="characteristicInput"
+              onChange={e => this.handleChange(e)} 
+              checked={this.state.tempCharacteristics[characteristicName] === this.characteristicDescriptions[characteristicName][3]}></input>
+            <input type="radio" name={characteristicName} value={4} className="characteristicInput"
+              onChange={e => this.handleChange(e)} 
+              checked={this.state.tempCharacteristics[characteristicName] === this.characteristicDescriptions[characteristicName][4]}></input>
+          </div>
+          <div className="formCharacteristicDescriptionBar">
+            <span className="formCharacteristicDescription">{this.characteristicDescriptions[characteristicName][0]}</span>
+            <span className="formCharacteristicDescription">{this.characteristicDescriptions[characteristicName][4]}</span>
+          </div>
+        </div>);
+      characteristics.push(characteristic);
+    }
```

Comment:
Love to see the templating, but it might be worthwhile to be defined outside of render so that the function itself isn't having to be declared every time the component re-renders.